pkg/storage/types.go

```diff
@@ -1,9 +1,12 @@
 package storage
 
 import (
+	"github.com/openshift/odo/pkg/machineoutput"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const StorageKind = "Storage"
```

Comment:
Will changing the kind from `storage` to `Storage` break the other tools using the json output?

Reply:
Yes, most probably. As the API of the JSON output is at the alpha level, it is expected